File: Assignment2/Assignment2_4.c
/////////////////////////////////////////////////////////////////
//Accept on character from user and check whether that character is vowel (a,e,i,o,u) or not. 
//Input : E Output : TRUE 
//Input : d Output : FALSE
////////////////////////////////////////////////////////////////

#include<stdio.h>
void main()
{
    char a;
    printf("Enter any letter between a-z or A-Z : ");
    scanf("%c",&a);
    printf("Your letter is : %c\n",a);

    if(a == 'a' || a == 'e' || a =='i' || a=='o' || a=='u' || a=='A' || a=='E' || a=='I' || a=='O' || a=='U')
    {
        printf("Given letter is vowel");
    }
    else
    {
        printf("Given letter is not vowel");
    }
}

File: Assignment2/Assignment2_4.py
#/////////////////////////////////////////////////////////////////
#//Accept on character from user and check whether that character is vowel (a,e,i,o,u) or not. 
#//Input : E Output : TRUE 
#//Input : d Output : FALSE
#////////////////////////////////////////////////////////////////

a = input("Enter any letter between a-z or A-Z : ")
    
print("Your letter is : ",a)

if(a == 'a' or a == 'e' or a =='i' or a=='o' or a=='u' or a=='A' or a=='E' or a=='I' or a=='O' or a=='U'):
        print("Given letter is vowel")
else:
        print("Given letter is not vowel")
    
